mayastor/src/grpc/mayastor_grpc.rs

```diff
@@ -228,17 +246,50 @@ impl mayastor_server::Mayastor for MayastorSvc {
                     return Err(Status::invalid_argument("Missing devices"));
                 }
 
-                let rx = rpc_submit::<_, _, LvsError>(async move {
-                    let pool = Lvs::create_or_import(args).await?;
-                    // Capture current pool config and export to file.
-                    PoolConfig::capture().export().await;
-                    Ok(Pool::from(pool))
-                })?;
-
-                rx.await
-                    .map_err(|_| Status::cancelled("cancelled"))?
-                    .map_err(Status::from)
-                    .map(Response::new)
+                let resp = match PoolBackend::try_from(args.pooltype)? {
+                    PoolBackend::Lvs => {
+                        if  MayastorFeatures::get_features().lvm {
+                            // check if a lvm pool already exists with the same name
+                            if let Some(_pool) = lookup_lvm_vg(args.name.as_str()).await {
+                                return Err(Status::invalid_argument("lvm pool with the same name already exists"))
```

Comment:
Can you please add pool name in the error message(s), that would simplify analysing error logs, as it clearly shows what name is conflicting.